controls/radimageeditor/getting-started.md

```diff
@@ -10,26 +10,94 @@ position: 1
 
-* Telerik.Windows.Controls;
+* __Telerik.Windows.Controls.ImageEditor__
 
-* Telerik.Windows.Controls.ImageEditor;
+* __Telerik.Windows.Controls.Input__
 
-* Telerik.Windows.Controls.Input.
+## Adding RadImageEditor to the Project
 
-The control offers a set of operations you can perform on an image which has been loaded in the editor. These operations are available as Commands and Tools, which you can execute both in code-behind or XAML. [Read More]({%slug radimageeditor-features-commands-and-tools%})
+The next few code examples will demonstrate how to add a __RadImageEditor__ in XAML, load a sample picture and execute a command on that picture. __Example 1__ showcases a __RadImageEditor__ and a Button defined in XAML.
 
-Adding the above listed references will be sufficient to enable different modifications on images using RadImageEditor’s API. If you would like to provide the end users with the ability to edit the images, too, then UI backing up the different modification must be added.
+#### __[XAML] Example 1: Defining a RadImageEditor in xaml__
 
-__RadImageEditor__ has a good-to-go UI that comes out of the box. That is __RadImageEditorUI__, which is quite easily wired to work with the commands and tools that __RadImageEditor__ exposes. As both controls follow closely the command pattern, they can be set up to work with little to no code-behind. However, you can implement and wire custom UI, too.  [Read More]({%slug radimageeditor-features-radimageeditorui%})
+{{region xaml-radimageeditor-getting-started-0}}
+	<Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition />
+            <RowDefinition Height="Auto" />
+        </Grid.RowDefinitions>
+        <telerik:RadImageEditor x:Name="ImageEditor"/>
+        <Button Click="Button_Click" Content="Rotate picture" Grid.Row="1" />
+    </Grid>
+{{endregion}}
+
+__Example 2__ shows the telerik namespace used in __Example 1__:
+
+#### __[XAML] Example 2: Telerik Namespace declaration__
+{{region xaml-radimageeditor-getting-started-1}}
+	xmlns:telerik="http://schemas.telerik.com/2008/xaml/presentation" 
+{{endregion}}
+
+In order to show a picture, you can set the __Image__ property of the __RadImageEditor__. It is of type [RadBitmap](https://docs.telerik.com/devtools/wpf/api/telerik.windows.media.imaging.radbitmap). __Example 3__ demonstrates how you can use the [ImageExampleHelper](https://github.com/telerik/xaml-sdk/blob/master/ImageEditor/RadImageEditorUIFirstLook/ImageExampleHelper.cs) class in order to load an Image. It assumes that there is a folder named "SampleImages" with an image named "RadImageEditor.png" inside the project. 
+
+#### __[C#] Example 3: Load image in RadImageEditor__
+
+{{region cs-radimageeditor-getting-started-2}}
+    public partial class MainWindow : Window
+    {
+        public MainWindow()
+        {
+            InitializeComponent();
+
+            ImageExampleHelper.LoadSampleImage(this.ImageEditor, "RadImageEditor.png");

```

Comment:
If I'm not mistaken, the ImageExampleHelper is a custom class and is not provided out of the box. In such case we should also include its implementation in this article.

Reply:
Apologies, just saw the above link. I still think that we should also copy the class in this article in case the customers also miss it.